config/crd/bases/expansion.gatekeeper.sh_expansiontemplate.yaml

```diff
@@ -79,6 +79,47 @@ spec:
                   generators, this is usually spec.template
                 type: string
             type: object
+          status:
+            description: ExpansionTemplateStatus defines the observed state of ExpansionStatus.
```

Comment:
@davis-haba can you pls run `make manfests` again to update the generated yamls.